modules/gltf/editor/editor_scene_importer_blend.cpp

```diff
@@ -516,6 +516,8 @@ void EditorFileSystemImportFormatSupportQueryBlend::_update_icons() {
 }
 
 bool EditorFileSystemImportFormatSupportQueryBlend::query() {
+	ERR_FAIL_COND_V_MSG(DisplayServer::get_singleton()->get_name() == "headless", true, "Blender is required to import .blend files but it could not be found.");
```

Comment:
What I mean is that this is not an error that blender cannot be found, it's that you're in headless mode, the error should reflect this, something like:
```suggestion
	ERR_FAIL_COND_V_MSG(DisplayServer::get_singleton()->get_name() == "headless", true, "Blender path is invalid or not set, check your Editor Settings. Cannot configure blender path in headless mode.");
```

Reply:
From the user's point of view they aren't trying to configure it so I think that makes it more confusing to say they "cannot" it. To them the error occurs because blender cannot be found, in a headless scenario this is likely because it isn't even installed, e.g. a CI build on a server.

Ultimately though any message with "blender" in it will be enough to point them in the right direction so let's run with it.